packages/react/src/components/DateTimePicker/DateTimePicker.jsx

```diff
@@ -556,6 +571,37 @@ const DateTimePicker = ({
     }
   };
 
+  const closeDropdown = useCloseDropdown({
+    isExpanded,
+    setIsCustomRange,
+    setIsExpanded,
+    parseDefaultValue,
+    defaultValue,
+    setCustomRangeKind,
+    dateTimePickerBaseValue,
+    setCurrentValue,
+    dateTimeMask,
+    toLabel: mergedI18n.toLabel,
+    setHumanValue,
+  });
+
+  useOnClickOutside(wrapperRef, closeDropdown);
+
+  // Close dropdown when focus leaves Apply button
+  const onDropdownBlur = (evt) => {
```

Comment:
This is needed because Apply button considered as the last focus state in the dropdown. When focus (tab) leaves Apply button then dropdown closes. For example, if 2 time pickers are near each other, user can use keyboard to open first dropdown, navigate trough it and jump from Apply button in the first dropdown to input of the second time picker. In this case, changes will be discarded and dropdown will close.

Reply:
@erzhan-temir-mamyrov I am confused about this one. A11Y users would need to use their keyboard to alter the calendar and time picker, etc. I would think that the dropdown traps focus, and you can't move your focus outside of it, but I don't think we should be closing it based on focus movement. That's not a behavior I've ever seen done elsewhere

Reply:
@JordanWSmith15 Thanks for the feedback! I will remove this behavior for focus movement.